### ISSUE TITLE
Share error "cannot write tempfile" because no initial "ask permission dialog" 

### ISSUE THREAD

**Issue by k3b** [guardianproject/ObscuraCam#133]
When you start the app for the first time via "gallery-image-select-share" and press the ObscuraCam-share-menu after modifying the image you get 

* no initial ask-writepermission-dialog from ObscuraCam
* a tooltip "cannot write tempfile" 

If you execute ObscuraCam-save for the first time

* the initial ask-writepermission-dialog from ObscuraCam is presented
* and save works as expected after granting permission.
* after this share also works as expected.

to reproduce: remove write-permission before test

**Comment by vArNi0101** [comment]
Same here, trying to share using "share"-button in the app gives tooltip error "saving temporary file failed". 

Storage permission is granted. Saving pictures works great. 

But it seems that there is no camera permission. No dialog asking for this permission.

Running Android 12 here.

**Comment by n8fr8** [comment]
Thanks for the update. Have you tried the beta release from earlier this year?

https://github.com/guardianproject/ObscuraCam/releases/tag/5.0.0-beta-1

We are just starting to focus on Android 12 and will work on an update soon.

**Comment by vArNi0101** [comment]
Yes tried this yesterday with a fresh installed version of 5.0.0-beta-1 from F-Droid
